playbooks/roles/kubernetes/README.md

```diff
@@ -0,0 +1,120 @@
+# Zowe Container Installation: Ansible Kubernetes Testing
+
+
+- [Zowe Container Installation: Ansible Kubernetes Testing](#zowe-container-installation-ansible-kubernetes-testing)
+  - [Introduction](#introduction)
+  - [Prerequisite](#prerequisite)
+  - [Environment Variables](#environment-variables)
+    - [More details about the environmental variables:](#more-details-about-the-environmental-variables)
+  - [Examples:](#examples)
+
+## Introduction
+
+This README covers the Zowe containers installation using Ansible Kubernetes Test. In this installation test, given the list of prerequisites in place, ansible first removes any Zowe containers deployed previous in the Kubernetes cluster and then it deploys all the Zowe containers to Kuberenetes cluster and wait till all of the pods become in Ready state. Since there are different Kubernetes platform, it is necessary to provide all the possible way to run the test using correct environment variables. To learn more about the Zowe container installation, please check [Installation Document](https://github.com/zowe/zowe-install-packaging/tree/master/containers/kubernetes)
+
+
+## Prerequisite
+
+In order to run this test, make sure that you have the required software and environments.
+
+1.	On the IBM Z – z/OS:
+•	Zowe installed on z/OS for users of ZSS and ZIS (default when you use the Zowe Application Framework app-server, the Zowe Desktop, or products that are based on them)
+•	z/OSMF installed on z/OS for users of it (default when you use gateway, API Mediation Layer, Web Explorers, or products that are based on them)
+2.	Kubernetes Cluster  - it could be installed/configured on BareMetal (zLinux/x86) or cloud (IBM Cloud Kubernetes, Google Cloud Kubernetes, OpenShift) 
+3.	Test Running System: Ansible – with module Kubernetes.core and kwoodson.yedit installed.
```

Comment:
Could we add few words how to get/install these Ansible modules?

Reply:
Sure. Thanks.